test/integration/task_polling_test.js

```diff
@@ -97,4 +98,43 @@ suite('Task Polling', () => {
       assert.equal(taskRes.run.reasonResolved, 'completed');
     });
   });
+
+  test('task polling at normal speed', async () => {
+    settings.billingCycleUptime(30);
+    settings.billingCycleInterval(40);
+
+    worker = new TestWorker(DockerWorker);
+    await worker.launch();
+    //make sure the polling timer is accurate
+    await base.testing.sleep(6000);
+
+    settings.billingCycleUptime(0);
+    await new Promise((accept, reject) => {
+      worker.on('polling', data => {
+        console.log(data.message);
```

Comment:
Also, regardless if we use waitForEvent or this promise, we don't want to be logging to console.log in tests and clouding up the test output.